README.md

```diff
@@ -1,24 +1,22 @@
 <p align="center">
   <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
-    <img src="https://github.com/aichner/React-MDB-Firebase-Template/blob/master/src/assets/react-mdb-firebase.jpg?raw=true" alt="Agency Logo" height="250" style="max-width: 100%" />
+    <img src="https://github.com/aichner/Company-Homepage/blob/master/src/assets/content/agency.jpg?raw=true" alt="Agency Logo" height="250" style="max-width: 100%" />
   </a>
 </p>
 
-<h3 align="center">Official React + MDB + Firebase Template</h3>
+<h3 align="center">Official Company Homepage</h3>
 
 <p align="center">
   This repository serves as an entry point for future React MDB Basic projects of the
   Advertisement Agency Christian Aichner.
```

Comment:
Please change the repository despcrion to:
`Open Source official company homepage of the Advertisement Agency Christian Aichner.`